css/style.css

```diff
@@ -0,0 +1,309 @@
+/* Global Styles */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+}
+
+a {
+    text-decoration: none;
+}
+
+body {
+    font-family: "Burbank Big Regular", sans-serif;
+    font-size: 18px;
+    line-height: 1.6;
+    /* Using Flexbox throughout the project */
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    font-family: "Burbank Big Regular", sans-serif;
+    text-transform: uppercase;
+}
+
+/* Main section 2 */
+.container {
+    box-sizing: border-box;
+}
+
+.wrapper {
+    padding: 16px 24px;
+}
+
+.flex {
+    display: flex;
+}
+
+.dividerOne {
+    height: 2px;
+    width: 100%;
+}
+
+.dividerTwo {
+    height: 24px;
+    width: 100%;
+}
+
+.card,
+.cardImgOne,
+.cardImgTwo,
+.cardImgThree,
+.cardImgFour {
+    box-shadow: 1px 1px 7px 0px #efe6e6;
+    border: 1px solid #f9f9f9;
+    border-radius: 4px;
+    background-color: #f9f9f9;
+    width: 100%;
+    flex-direction: column;
+}
+
+/* Two Big Fixed Images */
+.bigImages {
+    display: flex;
+    align-items: center;
+    flex-flow: row nowrap;
+    flex-direction: column;
+}
+
+.cardImgOne>.img {
+    height: 450px;
+    background: url("/images/bigImgOne.jpg");
```

Comment:
I can't see the imags
try doing this instead  background: url("../images/bigImgOne.jpg");

Reply:
This has been fixed a while ago and two dots before / worked. Thanks @ahmedserifoski :+1: